docs/design/uplink-scopes.md

```diff
@@ -93,16 +46,22 @@ paths := []BucketShare{
 }
 ```
 
-Then if one asked for `sj://x/a/d`, the 2nd path would be the longest match and `"key2"` would be used to encrypt `d` and it would be appened to `"/e1"`. Similarly, if one asked for `sj://x/a/b/c/f`, then `"key2"` would be used to encrypt `b/c/f` and it would be appened to `"/e1"`. If one asked for `sj://x/f/g`, then `"key3"` would be used. `"key1"` would never be used as it is subsumed by the entry with `"key2"`.
+Then if one asked for `sj://x/a/d`, the 2st path would be the longest match and `"key2"` would be used to encrypt `d` and it would be appened to `"/e1"`. Similarly, if one asked for `sj://x/a/b/c/f`, then the 1st path would be the longest match and `"key1"` would be used to encrypt `f` and it would be appened to `"/e1/e2/e3"`. If one asked for `sj://x/f/g`, then `"key3"` would be used. Asking for `sj://x/g` would return not found.
+
+Note that the encrypted path does not need to match the unencrypted path, and this allows for having local views of bucket paths in the actual bucket.
 
 ## Rationale
 
-This proposal is not a radical departure of the current method, just an extension to allowing a collection of paths to compose a bucket at any unencrypted spot. Thus, it shoudn't require any changes on the server side, reducing risk and allowing for earlier shipping.
+This proposal is not a radical departure of the current method, just an extension to allowing a collection of paths to compose a bucket at any unencrypted spot. Thus, it shoudn't require any changes on the server side, reducing risk and allowing for earlier shipping. For example, any current configuration
```

Comment:
A partial sentence was appended here. A mistake?

Reply:
Good catch. I'll complete the thought.